examples/src/pages/Migration.tsx

```diff
@@ -49,7 +49,9 @@ export function Migration() {
 
       const progress = (itemsLoaded: number, itemsRequested: number, itemsCulled: number) => {
         guiState.debug.loadedSectors.statistics.culledCount = itemsCulled;
-        console.log(`loaded ${itemsLoaded}/${itemsRequested} (culled: ${itemsCulled})`);
+        if (itemsLoaded === 0 || itemsLoaded === itemsRequested) {
+          console.log(`loaded ${itemsLoaded}/${itemsRequested} (culled: ${itemsCulled})`);
+        }
```

Comment:
Wasn't supposed to be part of this PR, but I'll leave it in. Basically this just reduces the spam in the Migration-example.